### PR TITLE
Added Java and Spring course

### PR DESCRIPTION
**Pull request by itsarvindh**
## What does this PR do?
Add resource(s)

### Description
Added the following courses

- Java Tutorial for Beginners Full course
- Spring Framework Complete Tutorial

### Why is this valuable (or not)?
I think it's valuable courses, as these courses demonstrate the teachings with very simple language. And I used these courses to learn few important concepts about Java and spring framework.

### How do we know it's really free?
It's hosted on YouTube and its completely free.

### For book lists, is it a book? For course lists, is it a course? etc.
It's a course.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by itsarvindh** [comment]
Hello, any updates on this PR ? As i can see a conflict with same playlist which was not there when i first made this PR !

**Comment by shrn01** [comment]
Please resolve the conflict and update the PR

**Comment by itsarvindh** [comment]
Conflicts merged, please check @eshellman @shrn01

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -666,6 +666,7 @@
 * [Java Server Tutorials - Happy Coding](https://happycoding.io/tutorials/java-server/) - Kevin Workman
 * [Java tutorial for beginners](https://youtube.com/playlist?list=PLZPZq0r_RZOMhCAyywfnYLlrjiVOkdAI1) - Bro Code (YouTube)
 * [Java Tutorial For Beginners](https://youtube.com/playlist?list=PLsyeobzWxl7oZ-fxDYkOToURHhMuWD1BK) - Telusko (YouTube)
+* [Java Tutorial for Beginners | Full Course](https://www.youtube.com/watch?v=8cm1x4bC610) - Navin Reddy, Telusko
```
Comment (shrn01) [inline courses/free-courses-en.md:669]: There is a lot of video overlap between previous entry and this entry as it is from the same creator but looks like lot of content is updated from the previous one. So this one would be a value add

**Review by shrn01 — APPROVED** [review]

File: courses/free-courses-en.md (hunk 1 above)
Comment (itsarvindh) [reply courses/free-courses-en.md:669]: Thank you @shrn01

Comment (eshellman) [reply courses/free-courses-en.md:669]: we prefer the playlist

**Pull request closed without merging**